brave/brave-core
# Wallet usage fix2

Resolves brave/brave-browser#21063.
Resolves brave/brave-browser#20956.

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

For usage ping, see brave/brave-browser#20956.

For P3A:

0) Set system date to Feb 11, 2022
1) Perform fresh install, start browser, and then close it.
2) Increment the system time by 9 days to skip the "no report" period for the weekly stat
3) Open the browser, use the wallet, close the browser.
4) Increment the system time by 1 day
5) Open the browser, use the wallet.
6) Open `brave://local-state`. Verify that the 
`Brave.Wallet.UsageDaysInWeek` P3A metric has a value of 2. Verify that the `Brave.Wallet.UsageMonthly.2` P3A metric has a value of 0. Close the browser
7) Set the system date to Mar 2, 2022
8) Open `brave://local-state`. Verify that the 
`Brave.Wallet.UsageDaysInWeek` P3A metric has a value of 0. Verify that the `Brave.Wallet.UsageMonthly.2` P3A metric has a value of 1.
